opencti-platform/opencti-front/src/utils/dashboard.ts

```diff
@@ -19,3 +21,10 @@ export interface DashboardWidgetParameters {
   legend?: boolean
   distributed?: boolean
 }
+
+export const copyPublicLinkUrl = (t: (text: string) => string, uriKey: string) => {
```

Comment:
Ok I think like your other PR it has a wrong push on the workspace/public dashboard stuff. 
I can approuve both of them, but can you please remove those unnecessary commits first please ? :)

Reply:
this is really weird 😕.
fixed